src/hooks/sprint-custom-hooks.ts

```diff
@@ -0,0 +1,93 @@
+import type { SelectChangeEvent } from "@mui/material";
+import { useState } from "react";
+import type { ResourceAllocations, ResourceAllocationsProject } from "src/generated/homeLambdasClient";
+
+const useSprintViewHandlers = () => {
+  const [filterType, setFilterType] = useState("project");
```

Comment:
Are there filter types other than project? If not then this is not needed, if there are you should create an enum to hold them e.g. SprintViewFilterTypes